Configuration.mk

```diff
@@ -88,9 +88,6 @@ override CPPFLAGS_cortex-m0 += $(CPPFLAGS_cortex-m) \
 override LEGACY_LIBS_cortex-m += \
       $(TOCK_USERLAND_BASE_DIR)/newlib/cortex-m/libc.a\
       $(TOCK_USERLAND_BASE_DIR)/newlib/cortex-m/libm.a\
-      $(TOCK_USERLAND_BASE_DIR)/libc++/cortex-m/libstdc++.a\
-      $(TOCK_USERLAND_BASE_DIR)/libc++/cortex-m/libsupc++.a\
-      $(TOCK_USERLAND_BASE_DIR)/libc++/cortex-m/libgcc.a
```

Comment:
Does this need to change for some reason?

Reply:
Oops, no that shouldn't be deleted.